pkg/cosign/cosign_test.go

```diff
@@ -33,20 +33,28 @@ const cosignPayload = `{
 	}
 }`
 
-const tektonPayload = `{
-  "Critical": {
-    "Identity": {
-      "docker-reference": "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/nop"
+const keylessPayload = `{
+    "critical": {
+      "identity": {
+        "docker-reference": "ghcr.io/jimbugwadia/pause2"
```

Comment:
Is there a Kyverno test image we can use instead?

Reply:
sure, we can use any image here